pkgs/by-name/py/pycdc/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pycdc";
+  version = "unstable-2024-10-13";
+
+  src = fetchFromGitHub {
+    owner = "zrax";
+    repo = "pycdc";
+    rev = "5e1c4037a96b966e4e6728c55b2d7ee8076a13c3";
+    hash = "sha256-c/mfM2I8Rw136aQ3IAQOkkrOEtZ5LC/xKuWXzCItW2w=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with lib; {
```

Comment:
`with lib;` has unexpected behaviors https://github.com/NixOS/nixpkgs/issues/208242

Reply:
Thanks, I was not aware of this undertaking (the things I packaged some time ago still use that syntax, but I understand it's an antipattern now)

Reply:
cant fix everything everywhere,  one package at the time :-)

Reply:
never heard about this one neither, thanks for the info!